packages/driver/cypress/e2e/cypress/error_utils.cy.ts

```diff
@@ -392,23 +392,16 @@ describe('driver/src/cypress/error_utils', () => {
 
     beforeEach(() => {
       $stackUtils.replacedStack = cy.stub().returns('replaced stack')
-      $stackUtils.stackWithUserInvocationStackSpliced = cy.stub().returns({ stack: 'spliced stack' })
       $stackUtils.getSourceStack = cy.stub().returns(sourceStack)
       $stackUtils.getCodeFrame = cy.stub().returns(codeFrame)
 
       err = { stack: 'Error: original stack message\n at originalStack (foo.js:1:1)' }
     })
 
-    it('replaces stack with user invocation stack', () => {
+    it('replaces stack with source map stack', () => {
       const result = $errUtils.enhanceStack({ err, userInvocationStack })
 
-      expect(result.stack).to.equal('replaced stack')
-    })
-
-    it('attaches source mapped stack', () => {
```

Comment:
Deleted this test since we're no longer attaching the property `sourceMappedStack`